api/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import postRouter from './routers/post.router';
+import dotenv from 'dotenv';
+import { internalServerErrorHandler, notFoundErrorHandler } from './middleware/errorHandler';
+
+const app = express();
+
+dotenv.config();
+
+const PORT = process.env.PORT;
+
+app.use('/', express.static('dist/test-project'));
+
+app.use('/api/posts', postRouter);
```

Comment:
Keep in mind, that new entities will require new paths, like /comments, /users etc. 
Better to move these into middle-level router like apiRouter and mount it to /api with /posts, /comments, /users